.github/workflows/ci-blenderbim-choco.yml

```diff
@@ -8,7 +8,7 @@ on:
     #         │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
     #         │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
     #         *  * * * *
-    - cron: "55 23 * * *" # 5min before utc midnight
+    - cron: "56 23 * * *" # 4min before utc midnight
```

Comment:
:)

Reply:
Heheh, yes. 🙂  
Idea is to have the window very small, where someone could publish 
another release before midnight, which would result in a diverging hash for choco.